setup.py

```diff
@@ -53,6 +54,11 @@
 
 LIBDATADOG_PROF_VERSION = "v2.1.0"
 
+UPX_VERSION = "v4.0.2"
+
+# Disable UPX compression when specified
+UPX_ENABLE = os.getenv("DD_UPX_ENABLE", "yes").lower() in ["false", "no"] and not DEBUG_COMPILE
```

Comment:
```suggestion
UPX_ENABLE = os.getenv("DD_UPX_ENABLE", "yes").lower() in {"false", "no", "0"} and not DEBUG_COMPILE
```
?

Reply:
Also checking that this is indeed the logic we want to implement. The default is `yes` but then we are assigning `True` to `UPX_ENABLE` if `DD_UPX_ENABLE` is falsy.